cvat/apps/engine/mixins.py

```diff
@@ -259,7 +259,7 @@ def export_annotations(self, request, pk, db_obj, export_func, callback, get_dat
             field_name=StorageType.TARGET,
         )
 
-        rq_id = "/api/{}/{}/annotations/{}".format(self._object.__class__.__name__.lower(), pk, format_name)
+        rq_id = f"api-{self._object.__class__.__name__.lower()}-{pk}-annotations-{format_name}"
```

Comment:
`export~annotations-for-{self._object.__class__.__name__.lower()}.id{pk}-in-{format_name}-format`